src/utils/schema/resolvers/Meta/getMetaPeopleResolver.ts

```diff
@@ -0,0 +1,51 @@
+import { Record } from '@models';
+// import { getPeople } from '@utils/proxy';
+
+/**
+ * Return people meta resolver.
+ *
+ * @param field field definition.
+ * @returns People resolver.
+ */
+const getMetaPeopleResolver = async (field: any) => {
+  const records = await Record.find({ resource: field.resource });
+  const peopleIds = [];
+  records.forEach((record) => {
+    const propertyValue = record.data[field.name];
+    peopleIds.push(...propertyValue.flat());
+  });
+  const getFilter = (people: any) => {
+    const formattedFilter = `{OR:[
+        userid_eq:
+        [${people.map((el: any) => `"${el}"`)}]`;
+    return formattedFilter.replace(/\s/g, '');
+  };
+  const filter = getFilter(peopleIds);
+  console.log(filter);
+
+  // const people = await getPeople(token, filter);
```

Comment:
@AntoineRelief Here is the problem discussed during the daily. The creation of a resolver may not be relevant in our case

Reply:
@NathanHGit 
what would be expected there
do a single query per record?

It's really a lot of requests to proceed yes

Reply:
@AntoineRelief No, the idea is to perform a single query that fetches only the selected records. Since there are more than 30.000 records, we decided to fetch only those that are being used. We send an array of ids as a filter to search for the corresponding users in the database. The filter looks like this:
`{OR: [{userid_eq: ["id1","id2]}]}`

Reply:
what I mean is that if you have a recordA and a recordB

recordA has id1, id2, recordB has the same ones

from what I see, you will do 2 different queries if you try to get the associated people, and the 2 queries would get the exact same users

Reply:
Yes, you are right. I see what you mean now